### PR TITLE
CCL-781: Fix-JS-SDK-doc

### PR DESCRIPTION
**Pull request by JohnJoser3**
Improve Clarity and ease of use.
Correct the outdated parts.

### CODE REVIEW

File: user-guide/docs/invoking-your-first-function-with-js-sdk.md [hunk 1]
```diff
@@ -1,192 +1,176 @@
...
-    accessToken
-});
+* Create an API key using the Conclave Cloud CLI command line tool:
+  `ccl apikeys create`
 
-export default conclaveConfig;
-```
+* Add the following configurations to `ConclaveCloud.js`
+   ```
+   export interface ConclaveClientConfig {
```
Comment (mivanovr3) [inline user-guide/docs/invoking-your-first-function-with-js-sdk.md:46]: This here will not work as is. It should be:

````
import { Conclave } from 'conclave-cloud-sdk'

const conclaveConfig = new Conclave.create({
    apiKey: 'apiKey' // place the apiKey you've created here
});
export default conclaveConfig;
````

**Review by mivanovr3 — CHANGES_REQUESTED** [review]
A change is required where the configuration is mentioned(posted a comment about it). Otherwise everything else looks good to me.

File: user-guide/docs/invoking-your-first-function-with-js-sdk.md (hunk 1 above)
Comment (JohnJoser3) [reply user-guide/docs/invoking-your-first-function-with-js-sdk.md:46]: Thanks Mihail!

**Review by mivanovr3 — APPROVED** [review]
LGTM. Approved.

**Pull request merged**